### ISSUE TITLE
CiviForm does not meet any a11y standard

### ISSUE THREAD

**Issue by cassandralfernandes** [seattle-uat/civiform#1941]
[An a11y audit in October '21](https://github.com/seattle-uat/civiform/issues/1639) documented 68 a11y problems of varying severity. Some of the most high-profile ones have been addressed but nearly all of them have not.

**Comment by kristinyim** [comment]
Lets close this and just use the issue already filed
